File: app/src/main/kotlin/de/ae/formulaecalendar/remote/pojo/calendar/SerieData.java
package de.ae.formulaecalendar.remote.pojo.calendar;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class SerieData {

    @SerializedName("Series")
    @Expose
    private String series;
    @SerializedName("Championship")
    @Expose
    private String championship;
    @SerializedName("ChampionshipId")
    @Expose
    private String championshipId;
    @SerializedName("Package")
    @Expose
    private String _package;
    @SerializedName("RaceCalendar")
    @Expose
    private RaceCalendar raceCalendar;
    @SerializedName("Generated")
    @Expose
    private String generated;

    public String getSeries() {
        return series;
    }

    public void setSeries(String series) {
        this.series = series;
    }

    public String getChampionship() {
        return championship;
    }

    public void setChampionship(String championship) {
        this.championship = championship;
    }

    public String getChampionshipId() {
        return championshipId;
    }

    public void setChampionshipId(String championshipId) {
        this.championshipId = championshipId;
    }

    public String getPackage() {
        return _package;
    }

    public void setPackage(String _package) {
        this._package = _package;
    }

    public RaceCalendar getRaceCalendar() {
        return raceCalendar;
    }

    public void setRaceCalendar(RaceCalendar raceCalendar) {
        this.raceCalendar = raceCalendar;
    }

    public String getGenerated() {
        return generated;
    }

    public void setGenerated(String generated) {
        this.generated = generated;
    }

    @Override
    public String toString() {
        return "SerieData{" +
                "series='" + series + '\'' +
                ", championship='" + championship + '\'' +
                ", championshipId='" + championshipId + '\'' +
                ", _package='" + _package + '\'' +
                ", raceCalendar=" + raceCalendar +
                ", generated='" + generated + '\'' +
                '}';
    }
}

File: formulaerest - Kopie/src/main/kotlin/de/ae/formulaecalendar/formulaerest/pojo/calendar/SerieData.kt
package de.ae.formulaecalendar.formulaerest.pojo.calendar

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class SerieData {

    @SerializedName("Series")
    @Expose
    var series: String? = null

    @SerializedName("Championship")
    @Expose
    var championship: String? = null

    @SerializedName("ChampionshipId")
    @Expose
    var championshipId: String? = null

    @SerializedName("Package")
    @Expose
    var `package`: String? = null

    @SerializedName("RaceCalendar")
    @Expose
    var raceCalendar: RaceCalendar? = null

    @SerializedName("Generated")
    @Expose
    var generated: String? = null

    override fun toString(): String {
        return "SerieData{" +
                "series='" + series + '\'' +
                ", championship='" + championship + '\'' +
                ", championshipId='" + championshipId + '\'' +
                ", _package='" + `package` + '\'' +
                ", raceCalendar=" + raceCalendar +
                ", generated='" + generated + '\'' +
                '}'
    }
}
